cmd/soroban-cli/src/commands/network/container/start.rs

```diff
@@ -82,17 +82,31 @@ impl Runner {
         let docker = self.args.container_args.connect_to_docker().await?;
 
         let image = self.get_image_name();
-        docker
-            .create_image(
-                Some(CreateImageOptions {
-                    from_image: image.clone(),
-                    ..Default::default()
-                }),
-                None,
-                None,
-            )
-            .try_collect::<Vec<_>>()
-            .await?;
+        let mut stream = docker.create_image(
+            Some(CreateImageOptions {
+                from_image: image.clone(),
+                ..Default::default()
+            }),
+            None,
+            None,
+        );
+
+        while let Some(result) = stream.try_next().await.transpose() {
+            if let Ok(item) = result {
+                if let Some(status) = item.status {
+                    if status.contains("Pulling from")
+                        || status.contains("Digest")
+                        || status.contains("Status")
+                    {
+                        self.print.infoln(status);
+                    }
+                }
+            } else {
+                self.print.warnln("Failed to fetch image from Docker Hub.");
```

Comment:
Could we inject the name of the image in this error instead of saying Docker Hub? Echoing bback the image being downloaded will help someone understand what isbeing pulled and failed.
```suggestion
                self.print.warnln("Failed to fetch image <image>.");
```

Reply:
Great idea! I'll get that change pushed up before merging